data/data/azure/main.tf

```diff
@@ -171,8 +171,8 @@ resource "azurerm_storage_blob" "rhcos_image" {
   resource_group_name    = azurerm_resource_group.main.name
   storage_account_name   = azurerm_storage_account.cluster.name
   storage_container_name = azurerm_storage_container.vhd.name
-  type                   = "block"
-  source_uri             = var.azure_image_url
+  type                   = "page"
+  source                 = var.azure_image_url
```

Comment:
@jomeier shouldn't this stay `source_uri` as before?

Reply:
No because the image will be decompressed locally. And for local sources this field must be 'source' instead of 'source_uri'.

Reply:
this may also be the reason then why the different type is needed?

Reply:
Even if I put a VHD file in a azure blob storage with Azure's web UI they tell you that for VHD files you should use page blob type. Page blobs can be accessed randomly and therefore multiple workers can upload/download from them in parallel. But if that's the reason: only speculations from my side.

Reply:
> this may also be the reason then why the different type is needed?

Ok